packages/conventional-changelog-angular/test/test.js

```diff
@@ -72,20 +74,26 @@ describe('angular preset', function () {
       .pipe(through(function (chunk) {
         chunk = chunk.toString()
 
+        except(chunk).to.include('first build setup')
```

Comment:
This needs to be changed from `except` to `expect` (And in the other locations below)

^ Minor typo there, which is why the [CI tests are failing](https://travis-ci.org/conventional-changelog/conventional-changelog/jobs/355569196#L2236).

Reply:
What about the `3) angular preset` fail? Is it also caused by this typo?